src/command_modules/azure-cli-batch/tests/batch_preparers.py

```diff
@@ -0,0 +1,70 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+from azure.cli.testsdk import CliTestError, ResourceGroupPreparer
+from azure.cli.testsdk.preparers import AbstractPreparer, SingleValueReplacer
+from azure.cli.testsdk.base import execute
+
+
+class BatchAccountPreparer(AbstractPreparer, SingleValueReplacer):
+    def __init__(self, name_prefix='clibatch',  # pylint: disable=too-many-arguments
```

Comment:
The `too-many-arguments` rule is disabled globally now. So you can remove the disable statement.